cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/RemoveTest.scala

```diff
@@ -53,16 +54,21 @@ class RemoveTest extends DocumentingTest with QueryStatisticsTestSupport {
           |Instead, if no value exists, the property is just not there.
           |So, `REMOVE` is used to remove a property value from a node or a relationship.""".stripMargin)
       query(
-        """MATCH (a {name: 'Andres'})
+        """MATCH (a {name: 'Andy'})
           |REMOVE a.age
           |RETURN a.name, a.age""".stripMargin, ResultAssertions((r) => {
-          r.toList should equal(List(Map("a.name" -> "Andres", "a.age" -> null)))
+          r.toList should equal(List(Map("a.name" -> "Andy", "a.age" -> null)))
           assertStats(r, propertiesWritten = 1, nodesDeleted = 0)
         })) {
         p("""The node is returned, and no property `age` exists on it.""".stripMargin)
         resultTable()
       }
     }
+    section("Remove all properties", "remove-remove-all-properties") {
+      p(
+        """`REMOVE` cannot be used to remove all existing properties from a graph element.
+          |Instead, using <<set-remove-properties-using-empty-map, `SET` with `=` and an empty map as the right operand>> will clear all properties from a node or element. """.stripMargin)
```

Comment:
"from a node or relatinoship" or "from a graph element". I prefere the former, I'm not used to "graph element". In fact I'd prefer "from a node or relationship." on line 69 as well.